doc/releases_and_migration/migration_guide_addon_v010.rst

```diff
@@ -146,6 +107,17 @@ If you encounter repository conflicts, do the following:
 
       west update
 
+Toolchain mismatch
+==================
```

Comment:
This should have a verb -ing form indicating the action performed at this step. Would this be correct: ?

```suggestion
Resolving toolchain mismatch
============================
```